<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphics_view">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>401</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="push_button_A_to_B">
       <property name="text">
        <string>A -&gt; B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_B_to_A">
       <property name="text">
        <string>B -&gt; A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_C_to_A">
       <property name="text">
        <string>C -&gt; A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time passed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Best time:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>401</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="push_button_A_to_C">
       <property name="text">
        <string>A -&gt; C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_B_to_C">
       <property name="text">
        <string>B -&gt; C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_C_to_B">
       <property name="text">
        <string>C -&gt; B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="text_browser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>71</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLCDNumber" name="lcd_number_seconds">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_number_minutes">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>71</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLCDNumber" name="lcd_number_best_seconds">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(78, 154, 6);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_number_best_minutes">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(78, 154, 6);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>65</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Seconds:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Minutes:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="text_browser_movement_history">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Movement history</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="push_button_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_reset_board">
       <property name="text">
        <string>Reset board</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_start_game">
       <property name="text">
        <string>Start game</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>push_button_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
